ga4gh/cli.py

```diff
@@ -196,6 +202,9 @@ def addGlobalOptions(parser):
     parser.add_argument(
         "--config-file", "-F", type=str,
         help="The configuration file to use")
+    parser.add_argument(
```

Comment:
Why do we want this? The reloader is generally pretty handy, and this server is just for development...